### ISSUE TITLE
Update Windows Tutorial

### ISSUE THREAD

**Issue by niccokunzmann** [schul-cloud/schul-cloud.github.io#71]
https://github.com/schul-cloud/schulcloud-server/pull/178 adds support for Windows seeding of the database.
The [Windows Tutorial](https://github.com/schul-cloud/schul-cloud.github.io/blob/master/dev/_posts/2017-05-18-setup-schul-cloud-client-and-sever-under-windows.md) should be adjusted.

@Langleu Do I read that right, that we can use same command in Windows and Linux to seed the database?

**Comment by Langleu** [comment]
yes, if you do as stated in this [comment](https://github.com/schul-cloud/schulcloud-server/pull/165#issuecomment-302483237)
